CHANGES.md

```diff
@@ -1,5 +1,8 @@
 ## Changes in 0.10.3 (in development)
 
+* Bands from cubes generated with `four_d` parameter set to true can be opened. 
+  (#101)
```

Comment:
Please describe the problem first. This reads as if the `four_d` parameter was first introduced to _avoid_ opening data cubes.